lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -208,7 +212,9 @@ spec = do
         "can perform roundtrip JSON serialization & deserialization, \
         \and match existing golden files" $ do
             jsonRoundtripAndGolden $ Proxy @(ApiAddress DummyTarget)
-            jsonRoundtripAndGolden $ Proxy @ApiBlockData
+            jsonRoundtripAndGolden $ Proxy @ApiTimeReference
+            jsonRoundtripAndGolden $ Proxy @ApiBlockReference
+            jsonRoundtripAndGolden $ Proxy @ApiNetworkInformation
```

Comment:
Should we have roundtrip tests for `NtpStatus` and `ProtocolUpdates`?

Reply:
Yes and no. We do already, since they're part of `ApiNetworkInformation`, but I guess it wouldn't hurt to also have them here.

Reply:
I think it would be worth adding them (if you haven't already), rather than relying on transitivity. There's always the possibility that someone will remove the compound types, and then we'd no longer be testing the atoms.